src/lib/isAfter.js

```diff
@@ -1,9 +1,11 @@
-import assertString from './util/assertString';
 import toDate from './toDate';
 
-export default function isAfter(str, date = String(new Date())) {
-  assertString(str);
-  const comparison = toDate(date);
-  const original = toDate(str);
-  return !!(original && comparison && original > comparison);
+export default function isAfter(date, options) {
+  // accessing 'arguments' for backwards compatibility: isAfter(str [, date])
+  // eslint-disable-next-line prefer-rest-params
+  const comparisonDate = (typeof options === 'object' ? options.comparisonDate : arguments[1]) || Date().toString();
```

Comment:
suggestion: I've just had an idea here to get rid of the ugly `// eslint-disable-next-line prefer-rest-params`:

```suggestion
export default function isBefore(date, ...args) {
    const [options] = args;
    const comparisonDate = (typeof options === 'object' ? options.comparisonDate : args[0]) || Date().toString();
    …
```

This is the more modern way of writing this and arguably more elegant (at the cost of an additional variable).

Reply:
You could go a step further:

```suggestion
  const comparisonDate = options?.comparisonDate || arguments[1] || Date().toString();
```

Reply:
Or just 

```suggestion
export default function isAfter(date, options) {
  // For backwards compatibility: isAfter(date [, comparisonDate]), i.e. `options` could be used as argument for the legacy `comparisonDate`
  const comparisonDate = options?.comparisonDate || options || Date().toString();
  const comparison = toDate(comparisonDate);
  const original = toDate(date);
```

Reply:
Good one! I've changed it